src/xpcc/architecture/platform/board/nucleo64_arduino.hpp

```diff
@@ -0,0 +1,30 @@
+// Nucleo64 Arduino Header Footprint
+
+#ifndef XPCC_STM32_NUCLEO_ARDUINO_HPP
+#define XPCC_STM32_NUCLEO_ARDUINO_HPP
+
+using A0 = GpioA0;
+using A1 = GpioA1;
+using A2 = GpioA4;
+using A3 = GpioB0;
+using A4 = GpioC1;
+using A5 = GpioC0;
+
+using D0  = GpioA3;
+using D1  = GpioA2;
+using D2  = GpioA10;
+using D3  = GpioB3;
+using D4  = GpioB5;
+using D5  = GpioB4;
+using D6  = GpioB10;
+using D7  = GpioA8;
+using D8  = GpioA9;
+using D9  = GpioC7;
+using D10 = GpioB6;
+using D11 = GpioA7;
+using D12 = GpioA6;
+using D13 = GpioA5;
+using D14 = GpioB9;
+using D15 = GpioB8;
```

Comment:
I was hesitant to do such a thing, since there is one Nucleo64 (F334) whose pinout differs in 3 pins. But lets solve it when it becomes an issue.